cloud-config.yml

```diff
@@ -96,12 +96,26 @@ bootcmd:
 - cloud-init-per once mkswap-swapfile mkswap /swapfile
```

Comment:
Should probably cross check with @lrowe 's https://github.com/ENCODE-DCC/encoded/pull/2652 for python 3.7/ES6+

Reply:
This should only change the cloud-config yaml.  lrowes didn't have any config/deploy like changes.